src/pages/RegisterPage/RegisterPage.tsx

```diff
@@ -1,8 +1,85 @@
+import { Box, Button, Link, TextField } from '@mui/material';
+import logo from '../../assets/logo/logo.svg';
+import { Link as RouterLink } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+
 const RegisterPage: React.FC = (): React.ReactElement => {
+  const [username, setUsername] = useState('');
```

Comment:
Why are we react-hook-form not being leveraged here? or  are we planning to refactor this and learn about it during the next meeting?

Reply:
I mentioned in the discord chat that I hadn't used react-hook-form yet and when we implement it we can change this